public/app/listeners/exportSpace.js

```diff
@@ -33,76 +34,111 @@ const validateConditions = (
   return spaceCondition && (studentCondition || visiblityCondition);
 };
 
+const prepareArchive = async (
+  archive,
+  db,
+  { space, spaceId, selection, userId }
+) => {
+  const {
+    space: isSpaceSelected,
+    actions: isActionsSelected,
+    resources: isResourcesSelected,
+  } = selection;
+
+  const isStudent = db.get('user.settings.studentMode').value();
+
+  // export the user's resources, private and public
+  const resources = isResourcesSelected
+    ? db
+        .get(APP_INSTANCE_RESOURCES_COLLECTION)
+        .filter((resource) =>
+          validateConditions(resource, spaceId, userId, isStudent)
+        )
+        .value()
+    : [];
+
+  const actions = isActionsSelected
+    ? db
+        .get(ACTIONS_COLLECTION)
+        .filter((action) =>
+          validateConditions(action, spaceId, userId, isStudent)
+        )
+        .value()
+    : [];
+
+  // copy prepackaged apps in space
+  if (isSpaceSelected) {
+    // get prepackaged apps
+    const isPrepackagedApp = (item) =>
```

Comment:
Why do prepackaged apps have these properties?

Reply:
`asset.startsWith(VAR_FOLDER)`?

Reply:
I check whether the item:
- is an app (category is `APPLICATION`)
- is referenced with an absolute link (since other assets will be available in the root of the space so do not refer to `VAR_FOLDER`). But I can indeed be more specific and check the apps folder.